### DISCUSSION TITLE
"at least" .DS_Store was added back

### DISCUSSION THREAD

**Discussion by arxanas** [jj-vcs/jj#88] [Q&A] [answered]
Got this message (in relation to #87):

```
$ jj untrack bin/.DS_Store 
Error: At least 'bin/.DS_Store' was added back because it is not ignored. Make sure it's ignored, then try again.
```

I can understand why `bin/.DS_Store` might have been added back, but why does it say "at least"? Under what circumstances would there be other files not listed here? (Also, it might be better wording to say `Error: 'bin/.DS_Store' (and possibly other files) were added back ...` as it's less of programmer-speak.)

**Comment by martinvonz** [comment, marked as answer]
Good point. We should probably just say exactly how many more files there are. It's not that expensive to do.

**Reply by martinvonz** [reply]
Oh, sorry, I should answer your question too :) The reason there may be more files is if you did something like `jj untrack some/dir/`, there may be many files in there that don't match the ignore patterns.
